pkg/ovs/ovn.go

```diff
@@ -33,13 +34,18 @@ type LegacyClient struct {
 	Version                       string
 }
 
-type ovnClient struct {
-	ovnNbClient
+type ovnNbClient struct {
+	ovsDbClient
 	ClusterRouter  string
 	NodeSwitchCIDR string
 }
 
-type ovnNbClient struct {
+type ovnSbClient struct {
+	ovsDbClient
+	NodeSwitchCIDR string
```

Comment:
Unused field.

Some fields defined in struct `LegacyClient` are also not used anymore, we should remove them.

Reply:
Error types `ErrNoAddr` and `ErrNotFound` are not used and they should be removed.

Reply:
Remove `OvnNbCtl` and `OvnSbCtl`.